### DISCUSSION TITLE
Finding a good layout - Operating Theatre Room

### DISCUSSION THREAD

**Discussion by ageekhere** [CorsixTH/CorsixTH#2440] [General Diagnostics]
I am exploring good design layouts for each room, in this post I am focusing on the Operating Theatre Room

**The Goal**
Keeping the room as small as possible while keeping the doctor as happy as possible(avoid being unhappy and wanting a pay rise) while optimizing turnaround time for handyman plant watering and patient walk times.

**Version 1.00**
![Screenshot 2023-10-16 212715](https://github.com/CorsixTH/CorsixTH/assets/13173133/ffc0a10e-19dc-4134-ab93-fdd91d2001d6)

Items
1x Operating Table
1x Surgeon Screen 
1x Sink
2x Fire Extinguisher
1x X-ray viewer
2x Radiator
1x Bin
1x Plant

**Summary**
The Operating Theatre is quite a complex room as there are many items that need to be placed in the room all which affect operating times. 

The idea with the Screen is to place it as close to the door as possible for fast changing. With the Sink both doctors need to wash their hands so creating a space behind the Operating Table and the skink allows them to achieve this while optimizing their walk path back to the Operating Table. X-ray viewer place as close to the doctor who is not performing the operation. The plant is not next to the door in this design. Bin is in the walk path of the patient. 2 Fire Extinguishers and 2 Radiator used to cover the large room.

Lets start the decision, can this design be improved and if so how?

**Comment by ravenPenfold** [comment]
You could improve the design by swapping the plant and fire extinguisher in the top corner, thus giving a extra path to the table in case the path is blocked, also giving the room slightly more space to move around.

**Comment by PaveMentman** [comment]
---

Ah yes, the "Operating Theatre".
For this, here is again one "inspiration" from my archives which most certainly isn't as optimized as the "Version 1.00"-image atop.
But it was still quite fast by giving enough walk-path-space for everyone while making a sacrifice of having an extra square or two needed to reach the points: 
![2023-08-28-10-10-08-404-CorsixTH-CorsixTH](https://github.com/CorsixTH/CorsixTH/assets/6709463/06dccf52-8e59-4084-80c4-243ce455a772) 

---

**Comment by ARGAMX** [comment]
Thanks, I like the idea of Finding a good layout for each room.

But why these optimisation conсentrated only for one cardinal direction entrance? 
How about others 3?

**Reply by Alberth289346** [reply]
These designs are for speed games, so completely optimized for maximum throughput.

That is a much stronger limit than "good layout", but you're of course free to give the other directions a try as well.

**Reply by ARGAMX** [reply]
> These designs are for speed games, so completely optimized for maximum throughput.
> 
> That is a much stronger limit than "good layout", but you're of course free to give the other directions a try as well.

I was considering this variant, but how does the trash bin increase the throughput in that case?

**Reply by ageekhere** [reply]
> > These designs are for speed games, so completely optimized for maximum throughput.
> > That is a much stronger limit than "good layout", but you're of course free to give the other directions a try as well.
> 
> I was considering this variant, but how does the trash bin increase the throughput in that case?

On the walk path to the operating tabled the trash bins allows the patient to drop off any trash they may have. This helps in the overall cleanness of the hospital.

**Reply by ghost** [reply]
Are plants even worth it for "optimum layout"?

**Reply by ageekhere** [reply]
> Are plants even worth it for "optimum layout"?

The idea is to have plants in the path of the staff to improve happiness, as for the number needed for the most optimum i am not sure.

**Comment by ARGAMX** [comment]
Do you use any instruments to plan and arrange items in room? 

How you get understanding which part of items is a passthrough and which are not?

**Reply by ageekhere** [reply]
I just watched the room and observed how the staff and patient walked around the room.

**Comment by Richard-L** [comment]
This is amazing. Would love more of such discussion. Need to try out the layout. Thanks for sharing!
